sdcm/cluster.py

```diff
@@ -2373,7 +2373,8 @@ def restart_node_with_resharding(self, murmur3_partitioner_ignore_msb_bits=12):
 
             return
 
-        resharding_finished = wait.wait_for(func=self._resharding_status, step=5,
+        # Decrease nodetool compactionstats calls from 5sec to 1min to avoid the noise
+        resharding_finished = wait.wait_for(func=self._resharding_status, step=60,
```

Comment:
maybe we should add some timeout, so it won't run indefinitely

Reply:
We discussed with Roy when I added this nemesis. Resharding process may take a while and we don't know how long it can be. So it was decided doesn't put timeout